tests/testsuite/cargo_add/git_inferred_name_multiple/stderr.log

```diff
@@ -1,2 +1,5 @@
     Updating git repository `[ROOTURL]/git-package`
-error: multiple packages found at `[ROOTURL]/git-package`: my-package1, my-package2
+error: multiple packages found at `[ROOTURL]/git-package`:
+    my-package1, my-package2, my-package3, my-package4, my-package5, my-package6
+    my-package7, my-package8, my-package9
+To disambiguate, run `cargo add --git [ROOTURL]/git-package <crate>`
```

Comment:
Should we also add a test for the error without any crate?

Reply:
> Should we also add a test for the error without any crate?

I'll add it.

> Also, I find it a little bit confusing to have the mix of two terms, `multiple (package)s found` and `<crate>` here.

Good catch! updated.

Reply:
Since this PR adds a new error path with splitting out the `path` case, I lean towards adding the tests in this PR

Reply:
Aha, do we really run into a situation where we can't even find a package? What happens when we can parse Cargo.toml but can't find any packages?

Reply:
I tested both path and git. It seems we have to find the `Cargo.toml` first. So I think it's an unreachable branch. So I just deleted it. You can check out https://github.com/rust-lang/cargo/pull/11186/commits/62bfdf0da16610f02450eb5a69ac4d6cdd93975a.